code/__DEFINES/wounds.dm

```diff
@@ -195,16 +201,19 @@ GLOBAL_LIST_INIT(wound_series_collections, generate_wound_series_collection())
 /// Allows for determining of which wound series are caused by what.
 GLOBAL_LIST_INIT(wounding_types_to_series, list(
 	WOUND_BLUNT = list(
-		WOUND_SERIES_BONE_BLUNT_BASIC
+		WOUND_SERIES_BONE_BLUNT_BASIC,
+		WOUND_SERIES_MUSCLE_DAMAGE // SKYRAT EDIT -- MUSCLE WOUNDS
```

Comment:
Always add trailing commas, it makes merge conflicts ever so slightly less likely to happen in the future.
```suggestion
		WOUND_SERIES_MUSCLE_DAMAGE, // SKYRAT EDIT -- MUSCLE WOUNDS
```